code/modules/mob/living/simple_animal/hostile/megafauna/drake.dm

```diff
@@ -39,31 +46,358 @@ Difficulty: Medium
 	armour_penetration = 40
 	melee_damage_lower = 40
 	melee_damage_upper = 40
-	speed = 1
-	move_to_delay = 10
-	ranged = 1
+	speed = 5
+	move_to_delay = 5
```

Comment:
Only significant divergance I can see is no GPS signal (Might or might not be a good thing?) for dragon and also we don't use slaughter demon attack sound for our drake.

Reply:
That seems... not right. The point of the GPS signals was to also help people who didn't want to go megafauna hunting and just mine in peace, right?

Reply:
@variableundefined the GPS is defined below, as `internal_type`; we don't have component based GPS like TG does, as such, it's an actual item in the mob itself.

Reply:
This is me being dumb and misreading